frontend/src/metabase/query_builder/actions/downloading.ts

```diff
@@ -168,9 +173,20 @@ const getDatasetResponse = ({
         formattedBody.append(key, JSON.stringify(body[key]));
       }
     }
-    return fetch(requestUrl, { method, body: formattedBody });
+    return POST(requestUrl, {
```

Comment:
Use `POST` and `GET` from `frontend/src/metabase/lib/api.js` as it takes care of the passing CSRF token.

I was considering a more static approach where I define the path upfront. But the path is not always static since we also call

https://github.com/metabase/metabase/blob/a14999fc7787cc4ac8bd6271376c1e240f047ceb/frontend/src/metabase/query_builder/actions/downloading.ts#L102